src/api.ts

```diff
@@ -63,7 +63,7 @@ export async function refreshLocalImages() {
 	const hasTag = (image: Docker.ImageInfo) => image.RepoTags.some( isTag );
 
 	state.localImages = new Map( images.filter( hasTag ).map( 
-		image => [ image.RepoTags[0], image ] as [ string, Docker.ImageInfo ]
+		image => [ image.RepoTags.find( isTag ), image ] as [ string, Docker.ImageInfo ]
```

Comment:
This could produce `[ undefined, image ]`

Reply:
thanks for calling that out. I'd like to address this in a larger project to understand how we're using images and containers. frankly I think we would become more reliable if we stopped depending on the `RepoTags` and instead linked via ids because as you caught and as we've seen we occasionally get builds with no tags.

in other words, thanks for catching this - I caught it too but I had no simple resolution that didn't involve an overhaul of how we refer to images. the bug will propagate the same way that it would propagate if we left it out - we wouldn't be able to reference that image.